install-scripts/install-debian.sh

```diff
@@ -0,0 +1,40 @@
+# This installation has been tested on Debian 11 XFCE
+
+# Install curl in order to install Rust and Cargo
+# Install make, necessary for installing python 3.9 with pyenv
+
+sudo apt install -y curl 
+                    \ make
```

Comment:
The backslash `\` must go in the previous line, for example:

```shell
sudo apt install -y curl \
                               make
```
its function is to escape the newline so the shell interprets it as a continuous command => `sudo apt install -y curl make`